### PR TITLE
Added ARIA Code hints

### PR DESCRIPTION
**Pull request by Coder206**
Now ARIA code hints are available in the drop down menu when coding, it worked well for me. Very intuitive like the other code hints. This is my first commit

### CODE REVIEW

**Comment by abose** [comment]
@Coder206 Thank you for contributing to Brackets. For discussions with the brackets community, please join the [brackets slack channel](https://brackets.slack.com/) by sending a mail to admin@brackets.io with the subject line `slack registration request` specifying the email addresses you would like to register.
[brackets dev google groups link here](https://groups.google.com/forum/#!forum/brackets-dev) .

**Comment by Coder206** [comment]
@abose Thank you.

**Comment by petetnt** [comment]
Review this, LGTM 👍

**Comment by RaymondLim** [comment]
@Coder206 Is there any reason or concern that you leave out [relationship attributes](https://www.w3.org/TR/wai-aria/states_and_properties#attrs_relationships)?

**Comment by petetnt** [comment]
@RaymondLim Good catch, I only validated those that were listed 👍

File: src/extensions/default/HTMLCodeHints/HtmlAttributes.json [hunk 1]
```diff
@@ -1,6 +1,38 @@
 {
-    "accesskey":          { "attribOption": [], "global": "true" },
-    "class":              { "attribOption": [], "global": "true", "type": "cssStyle" },
+    "accesskey":                { "attribOption": [], "global": "true" },
+    "aria-autocomplete":        { "attribOption": ["inline", "list", "both"]},
```
Comment (RaymondLim) [inline src/extensions/default/HTMLCodeHints/HtmlAttributes.json]: Default `none` is missing in value array.

**Comment by RaymondLim** [comment]
@Coder206 It will be nice if you can also add `role` attribute in this PR since it is also part of this ARIA spec.

**Comment by Coder206** [comment]
@RaymondLim I must have missed it when passing through the specification, I'll add them too. So you want value "none" as default in all of them? Do I place "role" inside "attribOption" like this '"attribOption": ["role"],'?

**Comment by RaymondLim** [comment]
@Coder206 Ignore my comment regarding `role` attribute. It is already in the file and no need to make any changes. `none` is only needed for the one that I comment on, not for every aria attribute. So the missing attributes that I noticed so far are `aria-activedescendant`, `aria-atomic` and `aria-flowto.

**Comment by Coder206** [comment]
@RaymondLim Ok perfect, thanks

**Comment by RaymondLim** [comment]
@Coder206 Based on the branch name `Coder206:patch-1` listed on this PR,  it seems like you're directly editing this file on GitHub. I'm not familiar with this kind of workflow and don't know how to make update to the same PR. @redmunds Do you know how @Coder206 can update his PR without having to create a different one?

**Comment by Coder206** [comment]
@RaymondLim Are you using the GitHub desktop app?

**Comment by redmunds** [comment]
@Coder206 @RaymondLim Directions for making changes to an existing branch using github website are here: https://github.com/adobe/brackets/tree/master/src/nls#updating-existing-branch-and-pull-request

**Comment by Coder206** [comment]
@redmunds Thanks will do this next time ;)

**Pull request closed without merging**